hummingbot/market/bittrex/bittrex_active_order_tracker.pyx

```diff
@@ -0,0 +1,163 @@
+# distutils: language=c++
```

Comment:
This whole file may be unneeded... if the diff messages we receive from Bittrex is already in (price, quantity) format, then you can simply use `apply_diff()` to update the order book. You can refer to the Binance market connector for how to do this - Binance doesn't have an active order tracker, it simply does `apply_diff()` in the order book tracker.